Example/package.json

```diff
@@ -11,44 +11,43 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
-    "@babel/plugin-proposal-export-namespace-from": "^7.17.12",
-    "@react-native-async-storage/async-storage": "^1.19.3",
-    "@react-native-community/slider": "^4.4.3",
-    "@react-native-masked-view/masked-view": "^0.3.0",
-    "@react-native-picker/picker": "^2.5.1",
-    "@react-navigation/native": "^6.1.8",
-    "@react-navigation/native-stack": "^6.9.14",
+    "@babel/plugin-transform-export-namespace-from": "^7.22.11",
+    "@react-native-async-storage/async-storage": "^1.21.0",
+    "@react-native-community/slider": "^4.5.0",
+    "@react-native-masked-view/masked-view": "^0.3.1",
+    "@react-native-picker/picker": "^2.6.1",
+    "@react-navigation/native": "^6.1.9",
+    "@react-navigation/native-stack": "^6.9.17",
     "react": "18.2.0",
-    "react-native": "0.72.6",
+    "react-native": "0.73.2",
     "react-native-box2d": "^0.2.5",
-    "react-native-gesture-handler": "^2.13.2",
-    "react-native-pager-view": "^5.4.25",
+    "react-native-gesture-handler": "^2.14.1",
+    "react-native-pager-view": "^6.2.3",
     "react-native-reanimated": "link:../",
-    "react-native-safe-area-context": "^4.7.2",
-    "react-native-screens": "^3.27.0",
-    "react-native-svg": "^13.14.0"
+    "react-native-safe-area-context": "^4.8.2",
+    "react-native-screens": "^3.29.0",
+    "react-native-svg": "^14.1.0"
```

Comment:
Thanks for bumping the libraries as well 🙏

Reply:
no problemo